libs/components/autonumeric/src/lib/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -52,103 +52,115 @@ export class SkyAutonumericDirective
    * Assigns the name of a property from `SkyAutonumericOptionsProvider`.
    */
   @Input()
-  public set skyAutonumeric(value: SkyAutonumericOptions) {
-    this.autonumericOptions = this.mergeOptions(value);
-    this.updateAutonumericInstance();
+  public set skyAutonumeric(value: SkyAutonumericOptions | undefined) {
+    this.#autonumericOptions = this.#mergeOptions(value);
+    this.#updateAutonumericInstance();
   }
 
   /**
    * @internal
    */
   @Input()
-  public skyAutonumericFormChangesUnformatted = false;
+  public skyAutonumericFormChangesUnformatted: boolean | undefined = false;
```

Comment:
Since this has a default value of `false` this should be converted to a getter/setter that sets `false` to a private variable internally whenever `undefined` is given and then the getter just returns that private variable.

Reply:
fixed in: [revert form control intialization form](https://github.com/blackbaud/skyux/pull/606/commits/7c5054fa10e9c64cd1386b7e6a725845e0659556)

Reply:
will you please confirm I did it the way you expected? thanks

Reply:
See Steve's content about using `??` instead of `||`.